internal/distro/rhel86/package_sets.go

```diff
@@ -450,8 +450,7 @@ func edgeCommitPackageSet(t *imageType) rpmmd.PackageSet {
 			"bash-completion", "tmux", "ima-evm-utils", "audit", "podman",
 			"container-selinux", "skopeo", "criu", "slirp4netns",
 			"fuse-overlayfs", "clevis", "clevis-dracut", "clevis-luks",
-			"greenboot", "greenboot-grub2", "greenboot-rpm-ostree-grub2",
-			"greenboot-reboot", "greenboot-status",
+			"greenboot", "greenboot-default-health-checks",
```

Comment:
don't we need the same change for rhel9's package_sets?

Reply:
or just requesting the old package names just redirect to the new one - nevermind (I'm testing it on a vm rn)

Reply:
yep, in 8.6 (and 9) requesting any of the old packages just gets you `greenboot` but I think none installs the new one (so this is needed even in 9)

Reply:
my original PR had both 8 and 9

Reply:
I changed the 9.0 back when I saw it wasn't in 9.0 yet thinking we might get this in for 8.6 only and do 9.0 later, but since it's coming in the next compose, let's keep it open and I'll put the change back (and redo the manifests) when we get it.